Src/Couchbase.Linq/QueryGeneration/N1QLQueryModelVisitor.cs

```diff
@@ -829,55 +830,55 @@ private N1QlFromQueryPart ParseNestClause(NestClause nestClause)
-                return VisitConstantExpressionNestClause(nestClause, nestClause.InnerSequence as ConstantExpression,
-                    GetExtentName(nestClause));
+                    return VisitConstantExpressionNestClause(nestClause, nestClause.InnerSequence as ConstantExpression,
+                        GetExtentName(nestClause));
             }
             else if (nestClause.InnerSequence is SubQueryExpression)
             {
                 var subQuery = (SubQueryExpression) nestClause.InnerSequence;
                 if (subQuery.QueryModel.ResultOperators.Any() ||
                     subQuery.QueryModel.MainFromClause.FromExpression.NodeType != ExpressionType.Constant)
-                {
-                    throw new NotSupportedException("Unsupported Nest Inner Sequence");
-                }
+                    {
+                        throw new NotSupportedException("Unsupported Nest Inner Sequence");
+                    }
 
-                // Generate a temporary item name to use on the NEST statement, which we can then reference in the LET statement
+                    // Generate a temporary item name to use on the NEST statement, which we can then reference in the LET statement
 
-                var genItemName = _queryGenerationContext.ExtentNameProvider.GetUnlinkedExtentName();
-                var fromPart = VisitConstantExpressionNestClause(nestClause,
-                    subQuery.QueryModel.MainFromClause.FromExpression as ConstantExpression, genItemName);
+                    var genItemName = _queryGenerationContext.ExtentNameProvider.GetUnlinkedExtentName();
+                    var fromPart = VisitConstantExpressionNestClause(nestClause,
+                        subQuery.QueryModel.MainFromClause.FromExpression as ConstantExpression, genItemName);
 
-                // Put any where clauses in the sub query in an ARRAY filtering clause using a LET statement
+                    // Put any where clauses in the sub query in an ARRAY filtering clause using a LET statement
 
-                var whereClauseString = string.Join(" AND ",
-                    subQuery.QueryModel.BodyClauses.OfType<WhereClause>()
-                        .Select(p => GetN1QlExpression(p.Predicate)));
+                    var whereClauseString = string.Join(" AND ",
+                        subQuery.QueryModel.BodyClauses.OfType<WhereClause>()
+                            .Select(p => GetN1QlExpression(p.Predicate)));
 
-                var letPart = new N1QlLetQueryPart()
-                {
-                    ItemName = GetExtentName(nestClause),
-                    Value =
-                        string.Format("ARRAY {0} FOR {0} IN {1} WHEN {2} END",
-                            GetExtentName(subQuery.QueryModel.MainFromClause),
-                            genItemName,
-                            whereClauseString)
-                };
+                    var letPart = new N1QlLetQueryPart()
+                    {
```

Comment:
@jeffrymorris 

Good catch, it was an artifact of the rebase to include the Relinq 2.0 work.  I've corrected it.

Thanks,
Brant